README.md

```
@@ -1,92 +1,121 @@
+![alt XOOPS CMS](https://xoops.org/images/logoXoops4GithubRepository.png)
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)